### ISSUE TITLE
Association of gut microbiota with the pathogenesis of SARS-CoV-2 Infection in people living with HIV

### ISSUE THREAD

**Issue by SvetlanaUP** [waldronlab/BugSigDBcuration#316]
Association of gut microbiota with the pathogenesis of SARS-CoV-2 Infection in people living with HIV – Aya Ishizaka – BMC Microbiology
https://bmcmicrobiol.biomedcentral.com/articles/10.1186/s12866-023-03157-5

**Comment by ardeybisi** [comment]
Good day, I would like to claim this article. 

Akinlagun Taiwo

**Comment by ardeybisi** [comment]
@SvetlanaUP I am done curating this paper: https://bugsigdb.org/Study_1026

**Comment by SvetlanaUP** [comment]
| Item | Description | max | points |
|---|---|---|---|
| 1 | All elements marked "Needs review" (none "Incomplete") | 1 | 0 |
| 2 | Correct study design | 1 | 0 |
| 3 | Entered all relevant experiments and no irrelevant experiments | 1 | 0 |
| 4 | Body site correctly identified (i.e. does not include multiple sites) | 1 | 1 |
| 5 | Condition entered according to contrast (correct EFO ontology) | 1 | 1 |
| 6 | Contrast groups correctly identified | 1 |0.5 |
| 7 | Groups correctly labeled as 1 and 0 (1=cases, 0=controls) | 1 | 0.5 |
| 8 | Antibiotic exclusion correctly identified | 1 | 1 |
| 9 | Correctly identified sequencing details | 2 | 2 |
| 10 | Identified correct statistical test | 1 | 0.5 |
| 11 | Identified MHT correction | 1 | 1 |
| 12 | Correctly recorded matched on factors | 1 | 0 |
| 13 | Entered correct number of statistical tests per experiment | 1 | 0.5 |
| 14 | All diversity measures identified | 1 | 0 |
| 15 | Diversity results correctly entered as increased/decreased/unchanged | 1 | 0.5 |
| 16 | All signature sources correctly identified (-1 for each error) | 2 | 1 |
| 17 | Abundance direction correctly selected | 1 | 1 |
| 18 | Members of signatures identified correctly | 2 | 0 |
| 19 | Correct use of NCBI taxonomy | 2 | 1 |
|  | TOTAL | 23 | 11.5 |

- Incomplete fields left (corrected)
- Incorrect study design (corrected)
- Confounders controlled for not present in the study (corrected)
- Observed OTUs (Richness) missed
- MaAsLin2 analysis missed
- Source of signature is Figure 2(C-E)
- There are 3 experiments

**Comment by SvetlanaUP** [comment]
[Nikilight](https://github.com/Nikilight) you could help out here in correction of this incomplete curation.

**Comment by Nikilight** [comment]
Yes @SvetlanaUP I will. Thank you for referring this to me.

**Comment by Nikilight** [comment]
Hello @SvetlanaUP I understand that you have been busy. I want to let you know that I have started correcting this curation, hence editing it on BugSigDB.

**Comment by SvetlanaUP** [comment]
@Nikilight great, just please make here notes what you have changed. Thanks!

**Comment by Nikilight** [comment]
Alright @SvetlanaUP will do.

**Comment by Nikilight** [comment]
Hello @SvetlanaUP, I have done the correction to the curation (https://bugsigdb.org/Study_1026) and is up for your review. Here is a list of all the changes I made:
1.	Included “Matched on” factor- Age.
2.	Included MaAsLin2 in the statistical Analysis.
3.	Included OTUs (Richness) in the Alpha Diversity
4.	Corrected the Signature sources and description (Figure 2C-E)
5.	Filled out incomplete fields such as the 2 other experiments.
6.	Corrected contrast group names (Group 0 and Group 1)
7.	Corrected group 0 sample size
8.	Edited all signatures to match with Figure 2C-E

I would like to be assigned another task if there are more. Thank you for the opportunity again.

**Comment by Scholarpat** [comment]
Hi @SvetlanaUP,

Here are the changes I made to this curation:

**- Condition:** HIV infection and SARS coronavirus were initially curated. Both study groups were people living with HIV (PLWH). Patients were compared based on the presence or absence of SARS-CoV-2 infection. I've changed this field to include only "SARS coronavirus."

**- Group names/definition:** The same group name/definition was curated for all three experiments. However, stool samples obtained from PLWH-CoV patients were classified into three groups according to the collection date: within seven days of onset (< 8 days); 8–14 days; and one month later. I've altered this field to reflect the different collection dates for clarity.

**- Sample size:** I've adjusted this to include the sample size that was used for the final analysis.

**- Confounder:** I've included body mass index (BMI) as a confounder since Multivariate Analysis with Linear Models (MaAsLin2) was used to adjust for BMI.

**- Matched-on:** 'Age' was initially curated, but the study did not report participants being matched on any variable. I edited this part out.

**- Alpha diversity:** Based on Figure 1B and this excerpt, richness and Shannon indices were unchanged for experiments 1-3; _"A comparison of observed operational taxonomic units (Observed OTU) showed no difference between HC and PLWH (Fig. 1B, left). In comparison to PLWH, an increase in Observed OTU values was observed from one week to one month after onset in PLWH-CoV, **but no statistically significant differences were found between them. Shannon index values did not differ among the three groups** (Fig. 1B, right)."_

**- Experiment 3, signature 2:** Changed 'Clostridiales bacterium' to 'Eubacteriales,' the heterotypic synonym of 'Clostridiales,' which was what was originally reported in the study.

**- Additional experiment/signatures:** PLWH with SARS-CoV-2 infection were further classified according to the disease severity into mild and moderate/severe. Differences were observed in the diversity of the bacterial microbiota only on the 8–14 days after the onset of SARS-CoV-2 infection. As such, patients were classified by the severity of their illness on days 8–14, and differences in bacterial microbiota profiles were analyzed again using LEfSe only in this group (Fig. 3E and F). I've curated this experiment with its signatures.

PS: Signature 2 of Experiment 4 is not displaying. I've reported this issue.

All changes have been effected, and the curation is complete.

Thank you!
